script/Arcueid/Day 10/10_00_ARC10_5C.txt

```diff
@@ -499,72 +532,83 @@ If that's what you say, then I won't talk about it any further. I won't ask for
 [sha:9d5b765e1be8028148ff5fbcd4b9f6ec81bfc443]{
 -- Page 540, Offset 25664.
 -- でも、ノエル先生との関係と、ノエル先生の背景については別問題だぞ。いいかげん、その代行者ってのが何なのか教えてくれ」
-But your relationship with Noel-sensei and her background are two different things. I need you to tell me about who those Executors are at the very least."
+// I swear to god if a whiteboard pops out I'm burning this file to the ground and salting the git repo
```

Comment:
I had the same thought and cried with joy when the file was over with nary a whiteboard
![image](https://user-images.githubusercontent.com/16092757/162867574-2bb09c91-974c-43bd-b5f8-766e661b1802.png)